sql/000-ore.sql

```diff
@@ -268,7 +284,19 @@ RETURNS integer AS $$
     cmp_result integer;
   BEGIN
     -- Recursively compare blocks bailing as soon as we can make a decision
-    RETURN compare_ore_array(a.terms, b.terms);
+    RETURN compare_ore_array(a.terms::bytea[], b.terms::bytea[]);
+  END
+$$ LANGUAGE plpgsql;
+
+DROP FUNCTION IF EXISTS compare_ore_64_8_v1(a bytea[], b bytea[]);
+
+CREATE FUNCTION compare_ore_64_8_v1(a bytea[], b bytea[])
+RETURNS integer AS $$
+  DECLARE
+    cmp_result integer;
+  BEGIN
+    -- Recursively compare blocks bailing as soon as we can make a decision
+    RETURN compare_ore_array(a, b);
```

Comment:
Bailing early can lead to timing attacks. I suspect that's why there is a variable (now unused) to keep the result while the rest of the traversal takes place.